app/javascript/leaflet/viewers/esri/tiled_map_layer.js

```diff
@@ -0,0 +1,73 @@
+import { tiledMapLayer, identifyFeatures } from "esri-leaflet";
```

Comment:
http://localhost:3000/catalog/nyu-test-soil-survey-map is now a feature layer, not a tiled map which means there is no way to test this, also I feel like if the info is in the esri like this https://dnrmaps.wi.gov/arcgis2/rest/services/TS_AGOL_STAGING_SERVICES/EN_AGOL_STAGING_SurfaceWater_WTM/MapServer/2 we should probably update it.